src/test/scala/no/ndla/imageapi/service/search/SearchServiceIntegrationTest.scala

```diff
@@ -0,0 +1,206 @@
+/*
```

Comment:
This is just a renaming of the file, in order to create a separate `SearchServiceTest` which is _not_ an integration test.